config.py

```diff
@@ -78,18 +78,58 @@
 
 # Social Links
 
-social_links: list[dict[str, str | bool]] = [
-    {"name": "Website", "url": "https://revanced.app", "preferred": True},
-    {"name": "GitHub", "url": "https://github.com/revanced", "preferred": False},
-    {"name": "Twitter", "url": "https://twitter.com/revancedapp", "preferred": False},
-    {"name": "Discord", "url": "https://revanced.app/discord", "preferred": True},
+social_links: list[dict[str, str, str, str | bool]] = [
+    {
+        "name": "Website",
+        "url": "https://revanced.app",
+        "icon_light": "https://revanced.app/favicon.ico",
+        "icon_dark": "https://revanced.app/favicon.ico",
+        "preferred": True,
+    },
+    {
+        "name": "Discord",
+        "url": "https://discord.gg/revanced",
+        # Icon is also at https://discord.com/assets/images/favicon.ico
+        # but that url fails to load from an Android embedded web view (unknown why).
```

Comment:
That's odd. Whats website-files.com? Pinging unknown domains may not be the best idea.